tests/suites/utils.js

```diff
@@ -309,12 +309,12 @@ casper.test.begin('isWebPage() tests', 3, function(test) {
     test.done();
 });
 
-casper.test.begin('isJsFile() tests', 5, function(test) {
+casper.test.begin('isJsFile() tests', 4, function(test) {
     var testCases = {
         '':             false,
         'toto.png':     false,
         'plop':         false,
-        'gniii.coffee': '.coffee' in require.extensions,
+//        'gniii.coffee': '.coffee' in require.extensions,
```

Comment:
@mickaelandrieu do you know how we process for a roll back ?

I will create a new PR meantime .